src/suit/js/SUIT.js

```diff
@@ -38,6 +39,9 @@ const lsstEntry= (label, url) => (
     {
         label,
         value: url,
+        fovDeg: 10,
+        centerWP: makeWorldPt(62,-37).toString(),
```

Comment:
I don't think the FOV and center are needed, unless there's evidence that the values in the deployed HiPS maps for DP0.2 are wrong.  Firefly should pick them up from there.  And even if that doesn't work right now, it would be better to patch it in deployment.

Reply:
The defines what the hips is centered on when the user click on the pointer by the target panel. I think you want this since the current data set is so limited.

Reply:
I thought our implementation respected `hips_initial_ra`, `hips_initial_dec`, and `hips_initial_fov` from the HiPS `properties` file for this

Reply:
I have not tested in the case, it might or not! But the direction of this to make is a finer grain setting than the hips dataset.

Reply:
Here's what we're currently providing:

<img width="398" alt="RSP-HiPS-initial" src="https://user-images.githubusercontent.com/13261203/177665610-3bb1e035-fac3-479b-a064-a9ffde516956.png">

If you want the smaller FOV, let's do that in the deployment files, not in the `suit` release.

Reply:
I will leave it like this for now. We might want to do further testing and iteration in the future